### DISCUSSION TITLE
Kaspersky flags the MSI installer as dangerous

### DISCUSSION THREAD

**Discussion by lordofscripts** [kanriapp/kanri#744] [Q&A]
Is it really safe? I downloaded the Windows version on an old laptop from my dad but Kaspersky blocks it.

**Comment by trobonox** [comment]
Apologies for not replying, it seems like I don't receive notifications for discussions opened in the organization but only those on the main repository.

According to VirusTotal, 0 vendors flagged the latest Kanri release (v0.8.0; the .msi variant) as malicious. 
As far as security is concerned, I try to regularly update all dependencies, so unless some library I use in the project got compromised it should be relatively safe (but do not take my word on this, I am not a security expert and you should feel free to take any security precautions you see suitable, e.g. running the app in a sandbox or cutting off its internet access).

Also are you sure you downloaded the MSI? I've ran into situations where the .exe gets flagged as malicious because under the hood it uses the NSIS installer which sometimes gets (falsely) picked up as malicious.
